packages/@o3r/workspace/schematics/application/schema.json

```diff
@@ -31,6 +31,16 @@
       "description": "Do not install dependency packages.",
       "type": "boolean",
       "default": false
+    },
+    "strict": {
+      "description": "Creates an application with stricter bundle budgets settings.",
+      "type": "boolean",
```

Comment:
I suppose the `$ref` to Angular Schematics schema.json was not working?

Reply:
indeed